compliance_checker/cf/cf_1_6.py

```diff
@@ -1864,8 +1865,7 @@ def check_time_coordinate(self, ds):
                     ret_val.append(result)
             # IMPLEMENTATION CONFORMANCE 4.4 RECOMMENDED 2/2
             # catch non-recommended months or years time interval
-            unit = Unit(variable.units)
-            if unit.is_long_time_interval():
```

Comment:
@benjwadams this is deprecated in cf-units and the change below should be implemented regardless of this PR. I'll open a new one just for this soon.